hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -279,12 +279,12 @@ public Hedera(
                         new AddressBookServiceImpl())
                 .forEach(servicesRegistry::register);
         try {
-            // And the factory for the MerkleHederaState class id must be our constructor
+            // And the factory for the MerkleStateRoot class id must be our constructor
             // FUTURE WORK: https://github.com/hashgraph/hedera-services/issues/11773
             constructableRegistry.registerConstructable(new ClassConstructorPair(
-                    MerkleHederaState.class, () -> new MerkleHederaState(new HederaLifecyclesImpl(this))));
+                    MerkleStateRoot.class, () -> new MerkleStateRoot(new MerkleStateLifecyclesImpl(this))));
         } catch (final ConstructableRegistryException e) {
-            logger.error("Failed to register MerkleHederaState factory with ConstructableRegistry", e);
+            logger.error("Failed to register MerkleStateRoot factory with ConstructableRegistry", e);
```

Comment:
```suggestion
            logger.error("Failed to register " + MerkleStateRoot.class + " factory with ConstructableRegistry", e);
```
This should not block the merge. Only fix this if another change is needed

Reply:
sounds good, thank you! I'll make this change a part of the next PR that will follow immediately after this one

Reply:
I fixed it here https://github.com/hashgraph/hedera-services/pull/14356/commits/ddf6a2eb7c9733a6db8c7efa97c05d4c5ed2866e